tests/safety/test_gm.py

```diff
@@ -196,6 +213,10 @@ def setUp(self):
     self.safety.set_safety_hooks(Panda.SAFETY_GM, Panda.FLAG_GM_HW_CAM)
     self.safety.init_tests()
 
+  # TODO: implement this
```

Comment:
@JasonJShuler Do you know what message this is?

Reply:
On cars that engage/disengage entirely based on PCM status rather than using buttons, I suspect this would be implemented.
Recall when you had to roll back the use of the boolean brake_pressed signal because the PCM uses a different threshold? The PCM would disengage over bumps and such, and I think it caused a controls mismatch or something...

In my opinion it warrants investigation: if cruise status is being managed by some ECU, I don't know why we aren't using the ECU state for engage/disengage, rather than buttons. (It might be related to improving the stop/go experience?)

Especially where these new cars will be initially dependent on the stock ACC...

Reply:
I think it was because we assume we can control the ACC state, including enabling and disabling without spamming buttons, and controlling the ACC set speed. For example we can't enable cruise on Toyota and we don't have cruise set speed control, but we can with Hyundai.

Do you know if the PCM manages these things or does the camera?